src/app/beer_garden/log.py

```diff
@@ -345,18 +365,17 @@ def _validate_formatters(formatters):
             return {"default": {"format": LoggingConfig.DEFAULT_FORMAT}}
 
         for formatter_name, _formatter_info in six.iteritems(formatters):
+            # TODO - WTF
```

Comment:
What needs to be done here?

Reply:
:laughing: 

It needs to be ripped out.